generator/internal/dart/dart.go

```diff
@@ -32,6 +32,8 @@ var dartTemplates embed.FS
 
 var typedDataImport = "dart:typed_data"
 var httpImport = "package:http/http.dart"
+var commonImport = "package:google_cloud_common/common.dart"
```

Comment:
Did we discuss moving these things to the configuration files? I do not recall if we opened a bug to reference here?

Reply:
From memory, I do need to pull the semver versions from configuration (we're hard-coded to `any` currently). I don't think this common import would go into configuration however - it's something that all generated packages will need.

I've been using https://github.com/googleapis/google-cloud-rust/issues/1034 to track work on the backend. It's a bigger, meta-issue. Now that there's a steel thread through I'm happy to refactor that into a few more stages. I could also break into separate issues, but I know this issue tracker is sync'd w/ a rust hostlist, so don't know that 1-2 dozen new dart issues would help things 🤷

Reply:
> From memory, I do need to pull the semver versions from configuration (we're hard-coded to `any` currently). I don't think this common import would go into configuration however - it's something that all generated packages will need.

Sold.

> I've been using #1034 to track work on the backend. It's a bigger, meta-issue. Now that there's a steel thread through I'm happy to refactor that into a few more stages. I could also break into separate issues, but I know this issue tracker is sync'd w/ a rust hostlist, so don't know that 1-2 dozen new dart issues would help things 🤷

Do not worry about the hotlist.